src/context.rs

```diff
@@ -1157,6 +1183,7 @@ mod tests {
         // too sensitive or summarized in another item.
         let skip_from_get_info = vec![
             "addr",
+            "delete_to_trash",
```

Comment:
My intention was to hide it from users, so they must read the source code to know about this config parameter. It can be set only with `set_config` command in CLI client, and if so, we can ask a user to show as the output of `get_config delete_to_trash`